kv/db_test.go

```diff
@@ -130,14 +130,15 @@ func TestKVDBInternalMethods(t *testing.T) {
 		args  proto.Request
 		reply proto.Response
 	}{
-		{&proto.InternalRangeLookupRequest{}, &proto.InternalRangeLookupResponse{}},
-		{&proto.InternalGCRequest{}, &proto.InternalGCResponse{}},
-		{&proto.InternalHeartbeatTxnRequest{}, &proto.InternalHeartbeatTxnResponse{}},
-		{&proto.InternalPushTxnRequest{}, &proto.InternalPushTxnResponse{}},
-		{&proto.InternalResolveIntentRequest{}, &proto.InternalResolveIntentResponse{}},
-		{&proto.InternalResolveIntentRangeRequest{}, &proto.InternalResolveIntentRangeResponse{}},
-		{&proto.InternalMergeRequest{}, &proto.InternalMergeResponse{}},
-		{&proto.InternalTruncateLogRequest{}, &proto.InternalTruncateLogResponse{}},
+		{&proto.HeartbeatTxnRequest{}, &proto.HeartbeatTxnResponse{}},
```

Comment:
yes, I think we should rip this out or at least disable it along with `internal`. If anything, we'll later check this against the user in a more central place (like `TxnCoordSender`).